docs/CHANGELOG.asciidoc

```diff
@@ -49,6 +49,36 @@ coming::[8.8.0]
 
 Review the following information about the {kib} 8.8.0 release.
 
+[float]
+[[known-issues-8.8.0]]
+=== Known issues
+
+// tag::known-issue-155203[]
+[discrete]
+.Monitors in Synthetics may stop running
+[%collapsible]
+====
+*Details* +
+If Monitor Management was enabled prior to 8.6.0, the API key generated internally will not contain the required permissions. The Synthetics app will attempt to fix this automatically in {kibana-pull}155203[#155203] when a user with sufficient privileges visits this page for the first time after upgrading to 8.8.0.
+
+*Impact* +
+All monitors configured to run on Elastic’s global managed testing infrastructure will stop running until a user with permissions has loaded the Synthetics app.
```

Comment:
Do you think it would be helpful to link to [the docs](https://www.elastic.co/guide/en/observability/8.8/synthetics-role-setup.html) that have more details on what "sufficient privileges" are?

Reply:
Good suggestion, added